utils/config.go

```diff
@@ -48,12 +48,13 @@ type (
 )
 
 func InitConfig() error {
-	user, err := getUser()
-	if err != nil {
-		return err
-	}
+	// user, err := getUser()
+	// if err != nil {
+	// 	return err
+	// }
 
-	homeDir := user.HomeDir
+	// homeDir := user.HomeDir
```

Comment:
Leaving this comment as a reminder to revert/find a fix for this.

Reply:
@sanganip Can use this to specify `--config-dir` https://github.com/cedana/cedana/pull/256 once merged.